website/docs/d/sm_arbitrary_secret.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_sm_arbitrary_secret (Beta)"
+description: |-
+  Get information about ArbitrarySecret
+subcategory: "IBM Cloud Secrets Manager API"
```

Comment:
Can we rename subcatergory from IBM Cloud Secrets Manager API to Secrets Manager so it displays in registry docs under services
![Uploading Screenshot 2023-02-13 at 11.08.22 AM.png…]()